contrib/!lang/typescript/packages.el

```diff
@@ -4,20 +4,62 @@
 ;; Copyright (c) 2014-2015 Chris Bowdon & Contributors
 ;;
 ;; Author: Chris Bowdon <c.bowdon@bath.edu>
+;; Author: Hamish Hutchings <hamhut1066@gmail.com>
 ;; URL: https://github.com/syl20bnr/spacemacs
 ;;
 ;; This file is not part of GNU Emacs.
 ;;
 ;;; License: GPLv3
 
-(setq typescript-packages '(tss))
+(setq typescript-private-packages '(
+                                    tss
+                                    flycheck))
 
-(defun typescript/init-tss ()
-  "Initialize my package"
+(setq test-layer-excluded-packages '(flymake))
```

Comment:
Is this necessary? `Flymake` is built in and doesn't interfere if its not activated